62_CAD/shaders/main_pipeline/vertex_shader.hlsl

```diff
@@ -706,19 +706,19 @@ PSInput vtxMain(uint vertexID : SV_VertexID)
 
             if (corner.x == 0.0f && corner.y == 0.0f)
             {
-                dilationVector.x = ieee754::flipSign(dilationVector.x);
+                dilationVector.x = ieee754::flipSign(dilationVector.x, true);
```

Comment:
you don't have to specify flip arg explicitly, you can cherry pick [this one](https://github.com/Devsh-Graphics-Programming/Nabla/commit/f334f90c1fc3b23d99f35e4cdf5227560a296954) or just change 2 LoCs according to the commit